dataline-scheduler/src/main/java/io/dataline/scheduler/WorkerRunFactory.java

```diff
@@ -71,9 +72,11 @@ public WorkerRunFactory(final Path workspaceRoot,
   }
 
   public WorkerRun create(final Job job) {
-    LOGGER.info("job: {} {} {}", job.getId(), job.getScope(), job.getConfig().getConfigType());
+    LOGGER.info("job id: {} attempt: {} scope: {} type: {}", job.getId(), job.getAttempts(), job.getScope(), job.getConfig().getConfigType());
```

Comment:
there's an "off-by-one" sort of thing here, where we update the number of attempts of a job in `JobSubmitter`, but the job that is passed into `WorkerRunFactory` is the job that was pulled from the db before this update happens. I think this is not a big deal, but if it bothers anyone we can do `getAttempts() + 1`  in here, or try to figure out how to inject the new job.

Reply:
They should match.